src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const NUMBER_OF_ATTEMPS = 3;
+  let counter = 0;
+
+  return (firstNumber) => {
+    return (secondNumber) => {
+      counter++;
+
+      if (counter > NUMBER_OF_ATTEMPS) {
+        counter -= 2;
+
+        return 'Bzzz... Error!';
```

Comment:
I think it would be better to make "'Bzzz... Error!'" into a constant, for example const ERROR_MESSAGE = 'Bzzz... Error!'

Reply:
➕